crew

```diff
@@ -20,25 +20,27 @@ DOC = <<DOCOPT
 Chromebrew - Package manager for Chrome OS http://skycocker.github.io/chromebrew/
 
 Usage:
-  crew build [-k|--keep] [-v|--verbose] <name> ...
-  crew const [<name> ...]
-  crew download [-v|--verbose] <name> ...
-  crew files <name> ...
+  crew build [options] [-k|--keep] <name> ...
+  crew const [options] [<name> ...]
+  crew download [options] <name> ...
+  crew files [options] <name> ...
   crew help [<command>]
-  crew install [-k|--keep] [-s|--build-from-source] [-S|--recursive-build] [-v|--verbose] <name> ...
-  crew list (available|installed|compatible|incompatible)
-  crew postinstall <name> ...
-  crew reinstall [-k|--keep] [-s|--build-from-source] [-S|--recursive-build] [-v|--verbose] <name> ...
-  crew remove [-v|--verbose] <name> ...
-  crew search [-v|--verbose] [<name> ...]
-  crew update
-  crew upgrade [-k|--keep] [-s|--build-from-source] [-v|--verbose] [<name> ...]
-  crew whatprovides <name> ...
+  crew install [options] [-k|--keep] [-s|--build-from-source] [-S|--recursive-build] <name> ...
+  crew list [options] (available|installed|compatible|incompatible)
+  crew postinstall [options] <name> ...
+  crew reinstall [options] [-k|--keep] [-s|--build-from-source] [-S|--recursive-build] <name> ...
+  crew remove [options] <name> ...
+  crew search [options] [<name> ...]
+  crew update [options]
+  crew upgrade [options] [-k|--keep] [-s|--build-from-source] [<name> ...]
+  crew whatprovides [options] <name> ...
```

Comment:
I'm not understanding how `[options]` takes care of `[-v|--verbose]`.

Reply:
In the release notes for the python version of docopt, there's a note about `[options]` that seems to hold true for the ruby version as well:

> New in version 0.6.0:
> ...
> * Meaning of [options] shortcut slightly changed. Previously it meant "any known option". Now it means "any option not in usage-pattern". This avoids the situation when an option is allowed to be repeated unintentionally.